website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -69,31 +69,31 @@ You also need to add another redirect URI to your Azure AD application. This red
 
 
 ## Connect Azure DevOps to your new app
-If you have already connected your Azure DevOps account to Active Directory, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Active Directory App you just created:
+If you have already connected your Azure DevOps account to Microsoft Entra ID, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Microsoft Entra ID App you just created:
 
 1. From your Azure DevOps account, select **Organization settings** in the bottom left.
-2. Navigate to Azure Active Directory.
+2. Navigate to Microsoft Entra ID.
 3. Click **Connect directory**.
 4. Select the directory you want to connect.
 5. Click **Connect**.
 
 <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/connect AD to Azure DevOps.gif" title="Connecting Azure DevOps and Active Directory"/>
 
-## Add your Azure AD app to dbt Cloud
+## Add your Microsoft Entra ID app to dbt Cloud
 
-Once you connect your Azure AD app and Azure DevOps, you need to provide dbt Cloud information about the app:
+Once you connect your Microsoft Entra ID app and Azure DevOps, you need to provide dbt Cloud information about the app:
 
 1. Navigate to your account settings in dbt Cloud.
 2. Select **Integrations**.
 3. Scroll to the Azure DevOps section.
 4. Complete the form:
     - **Azure DevOps Organization:** Must match the name of your Azure DevOps organization exactly. Do not include the `dev.azure.com/` prefix in this field. ✅ Use `my-devops-org` ❌ Avoid `dev.azure.com/my-devops-org`
-    - **Application (client) ID:** Found in the Azure AD App.
-    - **Client Secrets:** You need to first create a secret in the Azure AD App under **Client credentials**. Make sure to copy the **Value** field in the Azure AD App and paste it in the **Client Secret** field in dbt Cloud. You are responsible for the Azure AD app secret expiration and rotation.
-    - **Directory(tenant) ID:** Found in the Azure AD App.
+    - **Application (client) ID:** Found in the Microsoft Entra ID App.
+    - **Client Secrets:** You need to first create a secret in the Microsoft Entra ID App under **Client credentials**. Make sure to copy the **Value** field in the Microsoft Entra ID App and paste it in the **Client Secret** field in dbt Cloud. You are responsible for the Microsoft Entra ID app secret expiration and rotation.
+    - **Directory(tenant) ID:** Found in the Microsoft Entra ID App.
         <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/AzureDevopsAppdbtCloud.gif" title="Adding an Active Directory App to dbt Cloud"/>
```

Comment:
```suggestion
        <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/AzureDevopsAppdbtCloud.gif" title="Adding a Microsoft Entra ID App to dbt Cloud"/>
```

Reply:
nice catch! 

looked at the MS site, found https://learn.microsoft.com/en-us/entra/identity/enterprise-apps/overview-application-gallery , will mimic usage of "MS Entra app" (sans ID)